app/controllers/api/v1/events_controller.rb

```diff
@@ -25,54 +23,54 @@ def group_events
     end
 
     def add_movie
-      event = Event.find(params[:event_id])
+      event_1 = Event.find(params[:event_id])
```

Comment:
Yeah I could pass in the request body as a nested object but where would I be able to reuse the params? I'm only sending the event info through. Also, why now `event_1`? It's only scoped to that method so I don't see any conflict?